lib/ReactViews/DataCatalog/DataCatalogGroup.jsx

```diff
@@ -83,6 +90,8 @@ const DataCatalogGroup = createReactClass({
                 loading={group.isLoading}
                 emptyMessage="This group is empty"
                 onClick={this.clickGroup}
+                removable={canBeDeleted(this.props.group)}
+                removeUserAddedGroup ={this.removeUserAddedGroup}
```

Comment:
Actually, now that I see how this `removeUserAddedGroup` is used, I might get rid of it entirely. IMO it'd be fine if `CatalogGroup` knows how to remove a group from its parent (whatever it is) when `removable` is true and the user clicks that button.

Reply:
Rather than passing removeUserAddedGroup down as props, I mean.

Reply:
I think it's better to keep it in `DataCatalogGroup`  rather than passing `terria` and `group` into `CatalogGroup`? All the other actions are in DataCatalogGroup.